e2e/cypress/integration/archived_channel/archived_channel_spec.js

```diff
@@ -0,0 +1,139 @@
+
+    it('MM-T1678 Open an archived channel using CTRL/CMD+K', () => {
+        // # Select CTRL/CMD+K (or ⌘+K) to open the channel switcher
+        cy.visit(`/${testTeam.name}/channels/off-topic`);
+
+        cy.typeCmdOrCtrl().type('K', {release: true});
+
+        // # Start typing the name of a public or private channel on this team that has been archived
+        cy.findByRole('textbox', {name: 'quick switch input'}).type(testChannel.display_name);
+
+        // # Select an archived channel from the list
+        cy.get('#suggestionList').should('be.visible');
+        cy.findByTestId(testChannel.name).should('be.visible');
+        cy.findByTestId(testChannel.name).click();
+
+        // * Channel name visible in header
+        cy.get('#channelHeaderTitle').should('contain', testChannel.display_name);
+
+        // * Archived icon is visible in header
+        cy.get('#channelHeaderInfo .icon-archive-outline').should('be.visible');
+
+        // * Channel is listed In drawer
+        cy.get(`#sidebarItem_${testChannel.name}`).should('be.visible');
+        cy.get(`#sidebarItem_${testChannel.name} .icon-archive-outline`).should('be.visible');
+
+        // * footer shows you are viewing an archived channel
+        cy.get('#channelArchivedMessage').should('be.visible');
+    });
+
+    it('MM-T1679 Open an archived channel using jump from search results', () => {
+        // # Create or locate post in an archived channel where the test user had permissions to edit channel details
+        // generate a sufficiently large set of messages to make the toast appear
+        const messageList = Array.from({length: 40}, (_, i) => `${i}. any - ${getRandomId()}`);
+        createArchivedChannel({prefix: 'archived-search-for'}, messageList).then(({channelName}) => {
+            // # Locate the post in a search
+            cy.get('#searchBox').click().clear().type(`${messageList[1]}{enter}`);
+
+            // # Click jump to open an archive post in permalink view
+            cy.get('#searchContainer').should('be.visible');
+            cy.get('a.search-item__jump').first().click();
+
+            // * Archived channel is opened in permalink view
+            cy.get('.post--highlight').should('be.visible');
+
+            // # Click on You are viewing an archived channel.
+            cy.get('.toast__jump').should('be.visible').click();
+
+            // * Channel is listed In drawer
+            // * Channel name visible in header
+            cy.get(`#sidebarItem_${channelName}`).should('be.visible');
+            cy.get(`#sidebarItem_${channelName} .icon-archive-outline`).should('be.visible');
+
+            // * Archived icon is visible in header
+            cy.get('#channelHeaderInfo .icon-archive-outline').should('be.visible');
+
+            // * Footer shows "You are viewing an archived channel. New messages cannot be posted. Close Channel"
+            cy.get('#channelArchivedMessage').should('be.visible');
+        });
+    });
+});
```

Comment:
Tests here were moved as is and will be fixed once [MM-34298](https://mattermost.atlassian.net/browse/MM-34298) are resolved.